src/domains/dependencies/extractVersion.ts

```diff
@@ -0,0 +1,12 @@
+const versionRegExp = /(\d+)(\.\d+)(\.\d+)?/g;
+
+const extractVersion = (versionString: string): string => {
+  const regexResult = versionString.trim().match(versionRegExp);
+
+  if (!regexResult || !regexResult[0]) {
+    throw new Error('Invalid output');
```

Comment:
This message seems vague. Maybe:
```suggestion
    throw new Error(`Invalid version string: ${versionString}`);
```